process/guardian/guardedAccount.go

```diff
@@ -287,6 +287,19 @@ func (agc *guardedAccount) getActiveGuardian(gs *guardians.Guardians) (*guardian
 	return selectedGuardian, nil
 }
 
+// GetConfiguredGuardians returns the configured guardians for an account
+func (agc *guardedAccount) GetConfiguredGuardians(uah state.UserAccountHandler) (active *guardians.Guardian, pending *guardians.Guardian, err error) {
+	configuredGuardians, err := agc.getConfiguredGuardians(uah)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	active, _ = agc.getActiveGuardian(configuredGuardians)
```

Comment:
is it ok to ignore the error?

Reply:
This is a tough question
Is not having an active guardian an error?
Or not having a pending guardian an error?
or not having both?

Here I didn't consider the above errors, as these are configurations so preferences shouldn't be considered errors.
I am only returning an error in case the saved data is wrong.